### PR TITLE
OKTA-491325: Added documentation to get and update customizable template settings

### PR DESCRIPTION
**Pull request by chrisclare-okta**
## Description:
- **What's changed?**  Added docs related email template settings

- **Is this PR related to a Monolith release?** 2022.05.1 CF

### Resolves:

* [OKTA-491325)](https://oktainc.atlassian.net/browse/OKTA-491325)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 1]
```diff
@@ -2091,6 +2091,150 @@ HTTP/1.1 200 OK
...
+#### Response body
+
+The [Email Template Settings Object](#email-template-settings-object) resource that was updated.
+
+Returns a `422 Unprocessable Entity` if:
+- the template does not support recipients value.
+
+> **NOTE**: The templates that do not support all recipient values are the security templates, and the Change Email Confirmation template.
+
+Passing an invalid `brandId`, or `templateName`returns a `404 Not Found` with error code `E0000007`.
```
Comment (gormanho-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:2188]:
```suggestion
Passing an invalid `brandId` or `templateName` returns a `404 Not Found` with error code `E0000007`.
```

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 2]
```diff
@@ -2091,6 +2091,150 @@ HTTP/1.1 200 OK
 }
 ```
 
+
+### Email Template Settings
+
+This API lets you manage email template settings for each customizable email template.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:2097]: Redundant, I believe.
```suggestion
This API lets you manage the settings for each customizable email template.
```

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 3]
```diff
@@ -2091,6 +2091,150 @@ HTTP/1.1 200 OK
...
+
+#### Request body
+
+The [Email Template Settings Object](#email-template-settings-object) resource to update.
+
+#### Response body
+
+The [Email Template Settings Object](#email-template-settings-object) resource that was updated.
+
+Returns a `422 Unprocessable Entity` if:
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:2183]: If a list has one item we don't need the list :)
```suggestion
Returns a `422 Unprocessable Entity` if the template doesn't support the recipient's value.
```

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 4]
```diff
@@ -2091,6 +2091,150 @@ HTTP/1.1 200 OK
...
+- the template does not support recipients value.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:2184]:
```suggestion
```

**Review by gormanho-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 5]
```diff
@@ -2091,6 +2091,149 @@ HTTP/1.1 200 OK
...
+```http
+HTTP/1.1 200 OK
+```
+
+```json
+{
+    "recipients": "NO_USERS",
+    "_links": {
+        "template": {
+            "href": "https://${yourOktaDomain}/api/v1/brands/${brand}/templates/email/UserActivation",
```
Comment (anitadongare-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:2217]: maybe use the name ${brandId} in the links as well to keep it consistent with the request url on 2203?

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 6]
```diff
@@ -2091,6 +2091,149 @@ HTTP/1.1 200 OK
...
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            }
+        },
+        "self": {
+            "href": "https://${yourOktaDomain}/api/v1/brands/${brand}/templates/email/UserActivation/settings",
```
Comment (anitadongare-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:2225]: same ${brandId}

**Review by anitadongare-okta — APPROVED** [review]

**Review by brentschaus-okta — APPROVED** [review]
LGTM

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md (hunk 5 above)
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/reference/api/brands/index.md:2217]: @chrisclare-okta what do you think?

Comment (chrisclare-okta) [reply packages/@okta/vuepress-site/docs/reference/api/brands/index.md:2217]: Yup, this is correct. I'll change that now

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 7]
```diff
@@ -2091,6 +2091,154 @@ HTTP/1.1 200 OK
...
 
+### Email Template Settings
+
+<ApiLifecycle access="ea" />
+
+This API lets you manage the settings of each customizable email template.
+
+### Get Email Template Settings
+
+> **Note:** Email Template Settings are gated behind the **API For Supressing Email Notifications** Early Access feature must be enabled. See [Feature Lifecycle Management](https://developer.okta.com/docs/concepts/feature-lifecycle-management/) and [Manage Early Access and Beta features](https://help.okta.com/okta_help.htm?id=ext_Manage_Early_Access_features) for more information on Feature Manager.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:2102]:
```suggestion
> **Note:** Email Template Settings are gated behind the **API For Supressing Email Notifications** Early Access feature and must be enabled. See [Feature Lifecycle Management](/docs/concepts/feature-lifecycle-management/) and [Manage Early Access and Beta features](https://help.okta.com/okta_help.htm?id=ext_Manage_Early_Access_features) for more information.
```

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 8]
```diff
@@ -2091,6 +2091,154 @@ HTTP/1.1 200 OK
...
+| Parameter         | Type        | Description                  |
+| ----------------- | ----------- | ---------------------------- |
+| `brandId`         | String      | ID of a Brand                |
+| `templateName`    | String      | Name of an Email Template    |
+
+#### Response body
+
+The requested [Email Template Settings](#email-template-settings-object) resource, with variables populated using the current user's context.
+
+Passing an invalid `brandId`, or `templateName` returns a `404 Not Found` with error code `E0000007`.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:2120]:
```suggestion
Passing an invalid `brandId` or `templateName` returns a `404 Not Found` with error code `E0000007`.
```

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 9]
```diff
@@ -2091,6 +2091,154 @@ HTTP/1.1 200 OK
...
+#### Request path parameters
+
+| Parameter         | Type        | Description                  |
+| ----------------- | ----------- | ---------------------------- |
+| `brandId`         | String      | ID of a Brand                |
+| `templateName`    | String      | Name of an Email Template    |
+
+#### Response body
+
+The requested [Email Template Settings](#email-template-settings-object) resource, with variables populated using the current user's context.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:2118]:
```suggestion
The requested [Email Template Settings](#email-template-settings-object) resource, with variables populated using the current user's context
```

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 10]
```diff
@@ -2091,6 +2091,154 @@ HTTP/1.1 200 OK
...
+#### Request path parameters
+
+| Parameter         | Type        | Description                  |
+| ----------------- | ----------- | ---------------------------- |
+| `brandId`         | String      | ID of a Brand                |
+| `templateName`    | String      | Name of an Email Template    |
+
+#### Request body
+
+The [Email Template Settings Object](#email-template-settings-object) resource to update.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:2182]:
```suggestion
The [Email Template Settings Object](#email-template-settings-object) resource to update
```

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 11]
```diff
@@ -2091,6 +2091,154 @@ HTTP/1.1 200 OK
...
+| `brandId`         | String      | ID of a Brand                |
+| `templateName`    | String      | Name of an Email Template    |
+
+#### Request body
+
+The [Email Template Settings Object](#email-template-settings-object) resource to update
+
+#### Response body
+
+The [Email Template Settings Object](#email-template-settings-object) resource that was updated.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:2186]:
```suggestion
The [Email Template Settings Object](#email-template-settings-object) resource that was updated
```

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 12]
```diff
@@ -2091,6 +2091,154 @@ HTTP/1.1 200 OK
...
+
+The [Email Template Settings Object](#email-template-settings-object) resource to update
+
+#### Response body
+
+The [Email Template Settings Object](#email-template-settings-object) resource that was updated
+
+Returns a `422 Unprocessable Entity` if the template doesn't support the recipient's value.
+
+> **NOTE**: The templates that do not support all recipient values are New Sign-On Notification, Authenticator Enrolled, Authenticator Reset, Password Changed and the Change Email Confirmation templates.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:2190]:
```suggestion
> **NOTE**: The templates that don't support all recipient values are New Sign-On Notification, Authenticator Enrolled, Authenticator Reset, Password Changed, and Change Email Confirmation.
```

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 13]
```diff
@@ -2409,6 +2557,31 @@ The Email Customization resource defines the following properties:
 | `body`         | String                  | The body of the customization               |
 | `_links`       | [Links](#links-object)  | Link relations for this object              |
 
+<ApiLifecycle access="ea" />
+
+> **Note:** Email Template Settings are gated behind the **API For Supressing Email Notifications** Early Access feature must be enabled. See [Feature Lifecycle Management](https://developer.okta.com/docs/concepts/feature-lifecycle-management/) and [Manage Early Access and Beta features](https://help.okta.com/okta_help.htm?id=ext_Manage_Early_Access_features) for more information on Feature Manager.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:2562]:
```suggestion
> **Note:** Email Template Settings are gated behind the **API For Supressing Email Notifications** Early Access feature and must be enabled. See [Feature Lifecycle Management](/docs/concepts/feature-lifecycle-management/) and [Manage Early Access and Beta features](https://help.okta.com/okta_help.htm?id=ext_Manage_Early_Access_features) for more information.
```

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 14]
```diff
@@ -2409,6 +2557,33 @@ The Email Customization resource defines the following properties:
...
+| ------------------------------- | ----------------------------------------------- |
+| `ALL_USERS`                     | Send emails to all users (default)              |
+| `ADMINS_ONLY`                   | Send emails to administrators only              |
+| `NO_USERS`                      | Do not send emails to any user                  |
+
+
+> **Note:** The following list shows the email templates certain Enum values:
+> * `NO_USERS, ALL_USERS`: `NewSignOnNotification`, `PasswordChanged`, `AuthenticatorEnrolled`, and `AuthenticatorReset`
+> * `ALL_USERS, ADMIN ONLY`: `ChangeEmailConfirmation`
+The `ChangeEmailConfirmation` only allows following enum values: ALL_USERS, ADMIN ONLY
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:2585]:
```suggestion
```

**Pull request merged**